### DISCUSSION TITLE
Passing Variable via URL-Handler

### DISCUSSION THREAD

**Discussion by esamecar** [ttscoff/bunch-docs#171] [Q&A] [answered]
Hi, I have problems trying to pass a variable via URL-Handler. Can someone give me a hint what I am doing wrong?

To narrow my issue down, I've come up with this example. 

I open this URL via Safari: `x-bunch://open?bunch=timeryTest&startTimery=false`

I have a timeryTest.bunch and thought I could receive the variable "startTimery". So, is this supposed to work?:

```
---
title: Timery Test
toggles: false
ignore: false
---

(notify Variable is ${startTimery})
```

It just displays "Variable is", so startTimery is empty. Trying this

```
---
title: Timery Test
toggles: false
ignore: false
startTimery: true
---

(notify Variable is ${startTimery})
```

always displays "Variable is true" regardless of what I put into the URL. Orginally I was trying to pass the startTimery to a snippet and use conditional logic as in the Bunch help but that did not work, so I've tried this and wondering it this is a bug or not supposed to work like this ...

Thanks!

Marco

**Comment by ttscoff** [comment, marked as answer]
You're running into a case sensitivity bug, I think. I'll have to take 
another look at the code, but if you lowercase all references to the 
variable, I think just about everything you're trying here should work 
as documented.

Let me know if that helps, I'll track down the bug and get an update out 
soon.

-Brett

**Reply by esamecar** [reply]
that did the trick! thanks!

**Comment by ttscoff** [comment]
I've release version 1.4.7 with a fix for this issue. Thanks for reporting.
